### PR TITLE
Add simplistic Bioconductor sample.

### PR DESCRIPTION
**Pull request by deflaux**
Fixes https://github.com/googlegenomics/pipelines-api-examples/issues/4 with a minimum viable example.

### CODE REVIEW

File: bioconductor/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Use Bioconductor to count overlaps in BAM file
```
Comment (mbookman) [inline bioconductor/README.md]: "in *a* BAM file"?

File: bioconductor/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+* a [Bioconductor Docker container](https://bioconductor.org/help/docker/)
+* an [R script](./countOverlapsFromBAM.R)
+* and BAM file and index.
+
+It emits a one-line TSV file containing the count of overlapping reads and metadata identifying the sample from which the result was obtained.
+
+This simplistic example could be extended to:
+
+* run a more interesting Bioconductor analysis on the BAM file
```
Comment (mbookman) [inline bioconductor/README.md]: s/run/Run

File: bioconductor/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* loop over for example, all the 1000 Genomes phase 3 BAMs in [gs://genomics-public-data/ftp-trace.ncbi.nih.gov/1000genomes/ftp/phase3/data/\*/high_coverage_alignment/\*.bam](https://console.cloud.google.com/storage/browser/genomics-public-data/ftp-trace.ncbi.nih.gov/1000genomes/ftp/phase3/data/HG00096/high_coverage_alignment/), kicking off the parallel execution of this pipeline on each sample and emitting a distinct output file for each result.
```
Comment (mbookman) [inline bioconductor/README.md]: s/loop over for example/Loop over, for example,

File: bioconductor/run_bioconductor.py [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Python sample demonstrating use of the Google Genomics Pipelines API.
+
+This sample demonstrates a pipeline that uses Bioconductor to analyze
+files in Google Cloud Storage.
+
+This pipeline is run in an "ephemeral" manner; no call to pipelines.create()
+is neccessary. No pipeline is persisted in the pipelines list.
```
Comment (pgrosu) [inline bioconductor/run_bioconductor.py]: s/neccessary/necessary

File: bioconductor/run_bioconductor.py [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+"""
+
+import pprint
+import time
+
+from oauth2client.client import GoogleCredentials
+from apiclient.discovery import build
+
+PROJECT_ID='**FILL IN PROJECT ID**'
```
Comment (mbookman) [inline bioconductor/run_bioconductor.py]: Doesn't need to be changed now, but I'd like our examples to accept parameters.

File: bioconductor/README.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+```
+## (3) Upload the script to Cloud Storage.
+```
+gsutil cp countOverlapsFromBAM.R gs://YOUR-BUCKET/pipelines-api-examples/bioconductor/script.R
+```
+## (4) Run the pipeline on the cloud.
+Edit your copy of [run_bioconductor.py](./run_bioconductor.py) to specify:
+
+  1. the id of the Google Cloud Platform project in which the pipeline should run
+  1. the bucket in which the pipeline output file and log files should be placed.
```
Comment (mbookman) [inline bioconductor/README.md]: Should these two sentences both end in periods?

File: bioconductor/run_bioconductor.py (hunk 4 above)
Comment (mbookman) [inline bioconductor/run_bioconductor.py]: s/neccessary/necessary

File: bioconductor/run_bioconductor.py [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+            'https://www.googleapis.com/auth/compute',
+            'https://www.googleapis.com/auth/devstorage.full_control',
+            'https://www.googleapis.com/auth/genomics'
+        ]
+    }
+  }
+}).execute()
+
+# Emit the result of the pipeline run submission and poll for completion.
+POLL_INTERVAL = 20
```
Comment (mbookman) [inline bioconductor/run_bioconductor.py]: Move the POLL_INTERVAL to the top of the script.

**Comment by mbookman** [comment]
Just a few small things. Otherwise LGTM. Cool example.

**Comment by deflaux** [comment]
Fixed typos and filed https://github.com/googlegenomics/pipelines-api-examples/issues/12 for follow-up task.

**Pull request merged**